internal/ent/enums/auth_provider.go

```diff
@@ -3,25 +3,28 @@ package enums
 import (
 	"fmt"
 	"io"
+	"strings"
 )
 
 type AuthProvider string
 
 var (
 	// Credentials provider is when the user authenticates with a username and password
-	Credentials AuthProvider = "CREDENTIALS"
+	AuthProviderCredentials AuthProvider = "CREDENTIALS"
 	// Google oauth2 provider for authentication
-	Google AuthProvider = "GOOGLE"
+	AuthProviderGoogle AuthProvider = "GOOGLE"
 	// Github oauth2 provider for authentication
-	GitHub AuthProvider = "GITHUB"
+	AuthProviderGitHub AuthProvider = "GITHUB"
 	// Webauthn passkey provider for authentication
-	Webauthn AuthProvider = "WEBAUTHN"
+	AuthProviderWebauthn AuthProvider = "WEBAUTHN"
+	// AuthProviderInvalid is the default value for the AuthProvider enum
+	AuthProviderInvalid AuthProvider = "INVALID"
 )
 
 // Values returns a slice of strings that represents all the possible values of the AuthProvider enum.
 // Possible default values are "CREDENTIALS", "GOOGLE", "GITHUB", and "WEBAUTHN"
 func (AuthProvider) Values() (kinds []string) {
-	for _, s := range []AuthProvider{Credentials, Google, GitHub, Webauthn} {
+	for _, s := range []AuthProvider{AuthProviderCredentials, AuthProviderGoogle, AuthProviderGitHub, AuthProviderWebauthn, AuthProviderInvalid} {
```

Comment:
`AuthProviderInvalid` should not be included as a valid auth provider for the schema 

```suggestion
	for _, s := range []AuthProvider{AuthProviderCredentials, AuthProviderGoogle, AuthProviderGitHub, AuthProviderWebauthn} {
```